src/utilities/bcl/RemoteBCL.cpp

```diff
@@ -693,7 +728,7 @@ bool RemoteBCL::startComponentLibrarySearch(const std::string& searchTerm, const
 
   m_lastSearch.clear();
 
-  auto client = getClient(remoteUrl());
+  auto client = getClient(remoteUrl(), m_timeOutSeconds);
```

Comment:
Every internal call to getClient in RemoteBCL passes the member m_timeOutSeconds now